docs/sce_file_contents.md

```diff
@@ -0,0 +1,127 @@
+
+| Column name   | Contents                                                         |
+| ------------- | ---------------------------------------------------------------- |
+| `gene_symbol` | [HUGO](https://www.genenames.org) gene symbol, if defined        |
+| `mean`        | Mean count across all cells/droplets                             |
+| `detected`    | Number of cells in which the gene was detected (gene count > 0 ) |
+
+### Experiment metadata
+
+Metadata associated with {ref}`data processing <processing_information:Processing information>` is included in the `metadata` slot as a list.
+
+```r
+expt_metadata <- metadata(sce)
+```
+
+| Item name           | Contents                                                                                                                       |
+| ------------------- | ------------------------------------------------------------------------------------------------------------------------------ |
+| `salmon_version`    | Version of `salmon` used for initial mapping                                                                                   |
+| `reference_index`   | Transcriptome reference file used for mapping                                                                                  |
+| `total_reads`       | Total number of reads processed by `salmon`                                                                                    |
+| `mapped_reads`      | Number of reads successfully mapped                                                                                            |
+| `mapping_tool`      | Pipeline used for mapping and quantification (`alevin-fry` for all current data in ScPCA)                                      |
+| `alevinfry_version` | Version of `alevin-fry` used for mapping and quantification                                                                    |
+| `af_permit_type`    | `alevin-fry generate-permit-list` method used for filtering cell barcodes                                                      |
+| `af_resolution`     | `alevin-fry quant` resolution mode used                                                                                        |
+| `usa_mode`          | Boolean indicating whether quantification was done using `alevin-fry` USA mode                                                 |
+| `af_num_cells`      | Number of cells reported by `alevin-fry`                                                                                       |
+| `transcript_type`   | Transcripts included in gene counts: `spliced` for single-cell samples and `unspliced` for single-nuclei                       |
+| `miQC_model`        | The model object that `miQC` fit to the data and was used to calculate `prob_compromised`. Only present for `filtered` objects |
+
+
+## Additional SingleCellExperiment components for CITE-seq samples
```

Comment:
Do we want to put write SingleCellExperiment in code format as you are doing with the first header? I don't particularly have a preference, but we should be consistent on which way we choose to do it.

Reply:
I was going back and forth and forgot I did use code format before. I'll use that everywhere.